code/datums/traits.dm

```diff
@@ -234,9 +237,11 @@
 	var/disability_name = "" //! Name of the disability for medical records
 	var/disability_desc = "" //! Description of the disability for medical records
 	var/spawn_delay = 0 // To avoid ugly hardcoded spawn delay
+	var/list/mail_items = list() // Items to be sent via mail, const values require declaration in New()
 
 	New()
 		ASSERT(src.name)
+		src.setMail(src.mail_items)
```

Comment:
Ah no, use the `TYPEINFO_NEW` macro and initialize them there, otherwise you're still running the init for every trait datum instance.